turbo.json

```diff
@@ -154,6 +154,28 @@
         "!node_modules"
       ]
     },
+    "lint:fix": {
+      "inputs": [
+        "../../packages/eslint-config-custom/index.js",
+        "../../packages/eslint-config-custom/react.js",
+        "**/*.{ts,tsx}",
+        "!node_modules"
+      ]
+    },
+    "prettier": {
```

Comment:
Looks like we'll have a bunch of stuff that immediately starts failing when we merge this.
![image](https://github.com/user-attachments/assets/fc7d9f4b-b894-47c6-8db8-815c0f52d918)

Reply:
oh, yeah.... I guess we probably should ignore compiled files, eh? :sweat_smile:

Reply:
okay, updated to ignore any generated files and it seems to be passing now.

[update prettier ignore for generated files](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11095/commits/4bd5954bf13833a550cce4d572d903ab98732894)